Extension/src/nativeStrings.json

```diff
@@ -393,55 +393,60 @@
+        "text": "Invalid file path: '%s'.",
         "hint": "%s is the file that has an invalid path."
     },
     "e_cm_file_not_in_project": {
-        "text":  "File '%s' was not found.",
+        "text": "File '%s' was not found.",
         "hint": "%s is the file that was not found."
     },
-    "refactor_create_declaration_definition_failed":
-    {
+    "refactor_create_declaration_definition_failed": {
         "text": "Create Declaration / Definition failed: %s",
         "hint": "The operation 'Create Declaration / Definition' on a function was not successful. %s is the error info that has a period at the end of the string."
     },
-    "refactor_create_default_delete":
-    {
+    "refactor_create_default_delete": {
         "text": "Unable to create function '%s'. Creating defaulted or deleted functions is not supported.",
         "hint": "%s is the function that could not be created."
     },
     "refactor_function_copied_to_clipboard": "The function signature was copied to the clipboard.",
-    "refactor_function_not_created":
-    {
+    "refactor_function_not_created": {
         "text": "Unable to create function '%s'.",
         "hint": "%s is the function that could not be created."
     },
-    "refactor_ambiguous_locations":
-    {
+    "refactor_ambiguous_locations": {
         "text": "Unable to find an unambiguous location for function '%s'.",
         "hint": "%s is the function for the unambiguous location."
     },
-    "refactor_file_not_in_project":
-    {
+    "refactor_file_not_in_project": {
         "text": "File '%s' was not found.",
         "hint": "%s is the file that was not found."
     },
-    "refactor_not_class_namespace":
-    {
+    "refactor_not_class_namespace": {
         "text": "Could not find class or namespace '%s'.",
         "hint": "%s is the class or namespace code that could not be found."
     },
-    "refactor_operation_unsupported":
-    {
+    "refactor_operation_unsupported": {
         "text": "The operation is not supported for '%s'.",
         "hint": "%s is the function that is not supported for an operation that involves automatically generating code."
     },
     "unknown_error": "Unknown error.",
     "run_select_intellisense_configuration": {
         "text": "Please run the 'Select IntelliSense Configuration...' command to locate your system headers."
+    },
+    "offer_copy_declaration": {
+        "text": "Copy declaration of '{0}' to the clipboard",
+        "hint": "{0} is the name of a C/C++ function."
+    },
+    "offer_copy_definition": {
+        "text": "Copy definition of '{0}' to the clipboard",
+        "hint": "{0} is the name of a C/C++ function."
+    },
+    "refactor_copy_declaration_definition_failed": {
+        "text": "Copying Declaration / Definition to clipboard failed: %s",
+        "hint": "The operation 'Copy Declaration / Definition' on a function was not successful. %s is the error info that has a period at the end of the string."
```

Comment:
This 1st sentence of the hint seems unnecessary.

Reply:
...oh I see the same thing in other hints above. I'd prefer if those were removed too.

Reply:
Going to keep this for consistency.

Reply:
Sure, but couldn't both be removed for consistency too? Or do you mean consistency with something else?